### PR TITLE
Skeleton for blog post - r-universe interview

### PR DESCRIPTION
**Pull request by yabellini**

### CODE REVIEW

**Comment by yabellini** [comment]
@steffilazerte can you review?

Thing that are missing (for me):

- [ ] Add Spanish version when English is reviewed.
- [ ] Fix English video subtitles 
- [ ] Add correct link to video (after fix subtitles and made video public)

File: content/blog/2023-05-26-r-universe-stars-4/index.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+Their packages are organized into different groups where you can find basic packages, useful tools, and modules that include packages for data management and also packages to do statistical analysis on the inputs and outputs of the models.
+
+One of the most important points of[ having all the packages in an orderly manner in the R-Universe](https://pecanproject.r-universe.dev/builds) is to simplify the work of researchers. The models save them years of work and resources.
+
+In addition, with many people working on the packages, errors are more likely to decrease. Although the whole team knows that the PEcAn project code is not bug-free, they have a better chance of finding and fixing bugs by having more eyes looking at the same thing.
+
+In the following video, the team shows and tells how they organize the packages in the R Universe and what are the strengths of this tool:
+
+<embebed video>
```
Comment (steffilazerte) [inline content/blog/2023-05-26-r-universe-stars-4/index.md:66]:
```suggestion
```

**Review by steffilazerte — COMMENTED** [review]
Hi @yabellini, as always, it's so exciting to see what cool things people are doing with the R-Universe :grin:

I had one question about the reference to Google, as I didn't understand the context. Depending on that, I'll have a final suggestion for how to clarify that statement :grin: 

* [X] post follows Content Guidelines
* [X] post follows Style Guide
* [X] title is in Title Case (*with my suggestion*)
* [x] publication date is ok
* [X] alternative text of images is informative
* [X] author metadata is provided with correct folder name
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md
* [X] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)
* [X] YAML field 'preface' is present if necessary

File: content/blog/2023-05-26-r-universe-stars-4/index.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+An example of this practice is the project they are developing with Brooklyn National Laboratory, where they are working on NASA carbon monitoring that uses a data simulation system. In this case, two students are developing their PhDs and postdocs.
+
+## A tree that keeps growing 
+
+In this work structure, one of the main objectives is to become a kind of "Hacker" that solves problems in a more general sense that can then be applied in other contexts and in other areas. According to Rob, knowledge can be sustained and grow thanks to the research developed across multiple institutions including University of Illinois, Boston University, University of Arizona, University of Wisconsin and Brookhaven National Laboratory. 
+
+In relation to the experience of working together with Google, where they are looking to develop and promote ecosystem modeling tools, Chris explains that part of the success is that it is not just one university doing research but a large team of people all over the world working together. Although it takes longer, the result is used by a much wider community.
```
[the remark this answers is not in the document]
Comment (yabellini) [reply content/blog/2023-05-26-r-universe-stars-4/index.md:42]: In the video and the interview, they mention Google has supported them for seven years now.  Looking at their web page looks like Google Summer of Code.

File: content/blog/2023-05-26-r-universe-stars-4/index.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+By using standards and having packages that can solve the reading, embedding and writing of all the data involved in the process, researchers don't need to know how many different formats work, or communicate with many different research teams, in order to do this kind of work.  
+
+This allows an individual researcher to run a whole ensemble models. "Just a couple of years ago analyzing multi-models was not something that was accessible to individual researchers" NAME? explains.
```
Comment (yabellini) [inline content/blog/2023-05-26-r-universe-stars-4/index.md:53]:
```suggestion
This allows an individual researcher to run a whole ensemble models. "Just a couple of years ago analyzing multi-models was not something that was accessible to individual researchers" Michael explains.  
```

File: content/blog/2023-05-26-r-universe-stars-4/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (steffilazerte) [reply content/blog/2023-05-26-r-universe-stars-4/index.md:42]: Ah makes sense, what do you think of this edit? 
```suggestion
Working with Google Summer of Code, which has helped with developing and promoting an ecosystem of modeling tools, Chris explains that part of the success is that it is not just one university doing research but a large team of people all over the world working together. Although it takes longer, the result is used by a much wider community.
```

**Pull request merged**